### PR TITLE
core: refactor csv output

### PR DESCRIPTION
**Pull request by connorjclark**
example: https://docs.google.com/spreadsheets/d/13tkrlWJm4APDipB-KI63FOStIpObNyWL6PbYxil2aSU/edit?usp=sharing

image

### CODE REVIEW

File: report/generator/report-generator.js [hunk 1]
```diff
@@ -93,29 +93,51 @@ class ReportGenerator {
...
-    });
+        if (!audit) continue;
+
+        rows.push(rowFormatter([
+          category.id,
+          auditRef.id,
+          // TODO ?
+          // // CSV validator wants all scores to be numeric, use -1 for now
+          // const numericScore = audit.score === null ? -1 : audit.score;
+          audit.score || 'null',
```
Comment (adamraine) [inline report/generator/report-generator.js:131]: Problems for `audit.score === 0`?

Comment (connorjclark) [reply report/generator/report-generator.js:131]: yikes good catch

**Review by adamraine — APPROVED** [review]

**Review by bycorean — COMMENTED** [review]
approve

**Review by bycorean — COMMENTED** [review]
approve

**Pull request merged**